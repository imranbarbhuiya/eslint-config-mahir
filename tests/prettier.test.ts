import eslintConfig from '../_prettier';

describe('ESLint Config', () => {
	test('should export rules', () => {
		expect(eslintConfig).toMatchSnapshot();
	});
});
