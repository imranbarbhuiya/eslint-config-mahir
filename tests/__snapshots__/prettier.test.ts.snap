// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ESLint Config > should export rules 1`] = `
{
  "extends": [
    "prettier",
  ],
  "rules": {
    "arrow-body-style": 0,
    "line-comment-position": 0,
  },
}
`;
